cardinal/server/server.go

```diff
@@ -58,18 +59,27 @@ const (
 	getSignerForPersonaStatusAssigned  = "assigned"
 )
 
-// NewSwaggerHandler instantiates handler function for creating a swagger server that validates itself based on a swagger spec.
-func NewSwaggerHandler(w *ecs.World, pathToSwaggerSpec string, opts ...Option) (*Handler, error) {
+// NewHandler instantiates handler function for creating a swagger server that validates itself based on a swagger spec.
+func NewHandler(w *ecs.World, opts ...Option) (*Handler, error) {
+	h, err := newSwaggerHandlerEmbed(w, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return h, nil
+}
+
+//go:embed swagger.yml
+var swaggerData []byte
 
```

Comment:
The above is some reflection based thing that automatically pulls swagger.yml into swaggerData.